### DISCUSSION TITLE
rangeSlider should support more formatting functions

### DISCUSSION THREAD

**Discussion by gzzo** [algolia/instantsearch#5208] [Ideas]
The rangeSlider should support formatting for the min/medium/max values, similar to how it formats the tooltip.

This is how timestamps look currently:
![Date example](https://i.imgur.com/IHG4xxV.png)

**Do you want to request a *feature* or report a *bug*?**
Feature

**Feature: What is your use case for such a feature?**
Handling dates.  For example, a facet of timestamps.

**Feature: What is your proposed API entry? The new option to add? What is the behavior?**
Maybe something like

```
templates: {
   values: function(value) {
       return moment(value, 'X').format('ll');
   }
}
```

Where values refers to the min, medium, and max values shown under the rangeSlider.

**What is the version you are using? Always use the latest one before opening a bug issue.**
Latest

**Comment by bobylito** [comment]
Thanks for the feedback @gzzo This is not planned as we speak. In the mean time you could be interested in trying to implement your own slider using the [connector](https://community.algolia.com/instantsearch.js/v2/connectors/connectRange.html).
